docs/02-developers/05-smart-contracts/04-verify-smart-contracts/category.yml

```diff
@@ -0,0 +1,6 @@
+label: Verify Smart Contracts
```

Comment:
The file name is: `_category_.yml` containing these values you have here, this auto generates the index cards